builder/js/ncf.js

```diff
@@ -994,6 +995,12 @@ $scope.groupMethodsByCategory = function () {
     $scope.saving = false;
   });
 
+  $scope.newParam = {}
+  
+  $scope.addParameter = function() {
+    $scope.selectedTechnique.parameter.push(angular.copy($scope.newParam))
```

Comment:
I don't understand how and when newParam is modified

Reply:
In fact newParam does not need to exists anymore and could be replaced by an empty object to simplify things here